gits/test/capabilities_test.py

```diff
@@ -635,9 +635,12 @@ def test_cap_set_color(self):
     def test_cap_sgr0(self):
         pass
 
-    @unittest.skip('skip')
     def test_cap_op(self):
```

Comment:
Put the test between `test_cap_kcuu1` and `test_cap_ri` to arrange names in alphabetical order.